docs/data-ai/train/annotate-images.md

```diff
@@ -0,0 +1,472 @@
+---
+linkTitle: "Annotate images for training"
+title: "Annotate images for training"
+weight: 40
+layout: "docs"
+type: "docs"
+description: "Annotate images to train a machine learning model."
+---
+
+You can either manually add annotations through the Viam web UI, or add annotations with an existing ML model.
+
+## Prerequisites
+
+{{% expand "A machine connected to Viam" %}}
+
+{{% snippet "setup.md" %}}
+
+{{% /expand %}}
+
+{{% expand "A camera, connected to your machine, to capture images" %}}
+
+Follow the guide to configure a [webcam](/operate/reference/components/camera/webcam/) or similar [camera component](/operate/reference/components/camera/).
+
+{{% /expand %}}
+
+## Classify images with tags
+
+Classification determines a descriptive tag or set of tags for an image.
+For example, you could use classification to answer the following questions:
+
+- does an image of a food display appear `full`, `empty`, or `average`?
+- the quality of manufacturing output `good` or `bad`?
+- what combination of toppings exists on a pizza: `pepperoni`, `sausage`, and `pepper`? or `pineapple`, `ham`, and `mushroom`?
+
+Viam supports single and multiple label classification.
+To create a training dataset for classification, annotate tags to describe your images.
+
+{{< alert title="Tip" color="tip" >}}
+
+Unless you already have an ML model that can generate tags for your dataset, use the Web UI to annotate.
```

Comment:
this is confusing. Because it's sort of begs the question - so if I do have a model, what then? So it should link to that code. which maybe means that this should go to the annotate page: https://deploy-preview-4413--viam-docs.netlify.app/data-ai/train/update-dataset/#capture-annotate-and-add-images-to-a-dataset

Reply:
I think I misunderstand you, isn't this already on the annotate page? I put this admonition here to help guide people into the appropriate tab in the tabset that follows. Where are you thinking we could link?

Regardless I reworded this away from the question-begging 'unless', but me know if I'm missing something else.